catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/searchui/standard/endpoints/MetacardEditEndpoint.java

```diff
@@ -178,7 +175,7 @@ public Response setBinaryAttribute(@Context HttpServletResponse response,
         }
 
         catalogFramework.update(new UpdateRequestImpl(id, metacard));
-        Map<String, Object> responseMap = getResponseMap(attribute,
+        getResponseMap(attribute,
```

Comment:
This looks like a useless call to me if we aren't doing anything with the return value. Seems like maybe we just forgot to send this back in the response.